providers/aws/resources/aws.lr

```diff
@@ -1235,7 +1235,7 @@ private aws.dynamodb.table @defaults("name region") {
   // Tags for the table
   tags() map[string]string
   // The creation time of the table
-  createdTime time
+  createdAt time
```

Comment:
This is a breaking change of the resource. I think we need consensus first on how we structure those create and update timestamps. I am in full support to make this a consistent approach. We follow the API that we are exposing and they are not consistent either.